legend-engine-config/legend-engine-repl/legend-engine-repl-data-cube/pom.xml

```diff
@@ -74,86 +74,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <artifactId>maven-shade-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>shade</goal>
-                        </goals>
-                        <configuration>
-                            <shadedArtifactAttached>true</shadedArtifactAttached>
-                            <createDependencyReducedPom>false</createDependencyReducedPom>
-                            <filters>
-                                <filter>
-                                    <artifact>*:*</artifact>
-                                    <excludes>
-                                        <exclude>module-info.class</exclude>
-                                        <exclude>META-INF/**/module-info.class</exclude>
-                                        <exclude>META-INF/*.SF</exclude>
-                                        <exclude>META-INF/*.DSA</exclude>
-                                        <exclude>META-INF/*.RSA</exclude>
-                                    </excludes>
-                                </filter>
-                            </filters>
-                            <transformers>
-                                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
-                                <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
-                                    <mainClass>org.finos.legend.engine.repl.dataCube.client.DataCubeClient</mainClass>
-                                </transformer>
-                            </transformers>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
@akphi - would like to understand/discuss why we need to override default enforce rules here? 

IMO - It would be best to make sure we have a set of enforce rule all modules follow since will be hard to maintain and defeats the purpose if each module defines it own rules.

Reply:
This is a hack to make sure the shaded JAR when run does not complain about missing implementation for SLF4J, let's make sure the assembly we build does not have this issue. Ideally, I think the REPL should boot nicely without lots of Java warnings.

Reply:
My main goal right now is to avoid creating precedence on overriding the enforce rules.  

I will look into this on the next iteration, to make sure logging is properly configured without the need to bypass the enforce rules. 

With the assembly, we likely have some other options, like including a logback config to route logging to a file, etc rather than No-OP